### PR TITLE
Update README with docker distribution

### PR DESCRIPTION
**Pull request by jysohn23**
Instructions on how to consume docker distribution and train mnist on it with TPU.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,7 +124,24 @@ Then run `test/run_tests.sh` and `test/cpp/run_tests.sh` to verify the setup is
...
+Docker images with `torch` and `torch_xla` preinstalled in the `pytorch` conda
+environment are distributed under: `gcr.io/tpu-pytorch/xla`. This image has two type of tags which take the forms of:
+
+* gcr.io/tpu-pytorch/xla:nightly
+* gcr.io/tpu-pytorch/xla:YYYYMMDD (ex. gcr.io/tpu-pytorch/xla:nightly_20190531)
+
+With these images, for example, you can train mnist on TPUs with the following commands:
+
+```Shell
+docker pull gcr.io/tpu-pytorch/xla:nightly
```
Comment (dlibenzi) [inline README.md]: I would also add an line that shows how to run a shell within the Docker.
Many knows, but people that are new to it, they might be inclined to think that they need to run the docker every time.

Comment (jysohn23) [reply README.md]: Good idea! Done.

File: README.md [hunk 2]
```diff
@@ -122,7 +124,38 @@ Then run `test/run_tests.sh` and `test/cpp/run_tests.sh` to verify the setup is
...
+After pulling the image you can either:
+
+* Run the container with a command:
+```Shell
+docker run --shm-size 16G -e XRT_TPU_CONFIG="tpu_worker;0;<IP of the TPU node>:8470" gcr.io/tpu-pytorch/xla:nightly python pytorch/xla/test/test_train_mnist.py
+```
+
+* Run the script in an interactive shell:
+```Shell
+docker run -it --shm-size 16G gcr.io/tpu-pytorch/xla:nightly
```
Comment (dlibenzi) [inline README.md]: Is `/bin/sh` implicit?
I always passed it to `docker run -it ...`

Comment (jysohn23) [reply README.md]: I believe so, but it depends on the docker image's predefined entrypoint and debian's default I think is sh and we build off of that.

**Pull request merged**